keyboards/sirius/unigo66/info.json

```diff
@@ -0,0 +1,81 @@
+{
+	"keyboard_name": "UniGo66",
+	"url": "https://discord.gg/GJ8bdM",
+	"maintainer": "qmk",
+	"width": 17,
+	"height": 7,
+	"layouts": {
+		"LAYOUT": {
+			"layout": [
+				{"label":"Esc", "x":0, "y":0, "w":1.5},
+				{"label":"1", "x":1.5, "y":0},
+				{"label":"2", "x":2.5, "y":0},
+				{"label":"3", "x":3.5, "y":0},
+				{"label":"4", "x":4.5, "y":0},
+				{"label":"5", "x":5.5, "y":0},
+				{"label":"-", "x":6.5, "y":0},
+				{"label":"=", "x":9.5, "y":0},
+				{"label":"6", "x":10.5, "y":0},
+				{"label":"7", "x":11.5, "y":0},
+				{"label":"8", "x":12.5, "y":0},
+				{"label":"9", "x":13.5, "y":0},
+				{"label":"0", "x":14.5, "y":0},
+				{"label":"Bspc", "x":15.5, "y":0, "w":1.5},
+				{"label":"Tab", "x":0, "y":1, "w":1.5},
+				{"label":"Q", "x":1.5, "y":1},
+				{"label":"W", "x":2.5, "y":1},
+				{"label":"E", "x":3.5, "y":1},
+				{"label":"R", "x":4.5, "y":1},
+				{"label":"T", "x":5.5, "y":1},
+				{"label":"L1", "x":6.5, "y":1, "h":1.5},
+				{"label":"[", "x":9.5, "y":1, "h":1.5},
+				{"label":"Y", "x":10.5, "y":1},
+				{"label":"U", "x":11.5, "y":1},
+				{"label":"I", "x":12.5, "y":1},
+				{"label":"O", "x":13.5, "y":1},
+				{"label":"P", "x":14.5, "y":1},
+				{"label":"\\", "x":15.5, "y":1, "w":1.5},
+				{"label":"Caps", "x":0, "y":2, "w":1.5},
+				{"label":"A", "x":1.5, "y":2},
+				{"label":"S", "x":2.5, "y":2},
+				{"label":"D", "x":3.5, "y":2},
+				{"label":"F", "x":4.5, "y":2},
+				{"label":"G", "x":5.5, "y":2},
+				{"label":"L2", "x":6.5, "y":2.5, "h":1.5},
```

Comment:
The order of JSON objects doesn't match the order in `unigo66.h`, which will lead to misplaced keys if someone uses QMK Configurator to build their firmware.

There's a series of changes needed to fix this file.

```diff
diff --git a/keyboards/sirius/unigo66/info.json b/keyboards/sirius/unigo66/info.json
index 3a38bea74..250a87b63 100644
--- a/keyboards/sirius/unigo66/info.json
+++ b/keyboards/sirius/unigo66/info.json
@@ -41,8 +41,6 @@
 				{"label":"D", "x":3.5, "y":2},
 				{"label":"F", "x":4.5, "y":2},
 				{"label":"G", "x":5.5, "y":2},
-				{"label":"L2", "x":6.5, "y":2.5, "h":1.5},
-				{"label":"]", "x":9.5, "y":2.5, "h":1.5},
 				{"label":"H", "x":10.5, "y":2},
 				{"label":"J", "x":11.5, "y":2},
 				{"label":"K", "x":12.5, "y":2},
@@ -55,6 +53,8 @@
 				{"label":"C", "x":3.5, "y":3},
 				{"label":"V", "x":4.5, "y":3},
 				{"label":"B", "x":5.5, "y":3},
+				{"label":"L2", "x":6.5, "y":2.5, "h":1.5},
+				{"label":"]", "x":9.5, "y":2.5, "h":1.5},
 				{"label":"N", "x":10.5, "y":3},
 				{"label":"M", "x":11.5, "y":3},
 				{"label":",", "x":12.5, "y":3},
@@ -68,12 +68,12 @@
 				{"label":"Down", "x":14.5, "y":4},
 				{"label":"Right", "x":15.5, "y":4},
 				{"label":"Space1", "x":5.5, "y":5},
-				{"label":"Space2", "x":5.5, "y":6},
 				{"label":"PgUp", "x":6.5, "y":5},
-				{"label":"PgDn", "x":6.5, "y":6},
 				{"label":"Home", "x":9.5, "y":5},
-				{"label":"End", "x":9.5, "y":6},
 				{"label":"Enter1", "x":10.5, "y":5},
+				{"label":"Space2", "x":5.5, "y":6},
+				{"label":"PgDn", "x":6.5, "y":6},
+				{"label":"End", "x":9.5, "y":6},
 				{"label":"Enter2", "x":10.5, "y":6}
 			]
 		}
```